### ISSUE TITLE
Start and stop charging not working in 0.25.0

### ISSUE THREAD

**Issue by Preventlow** [tillsteinbach/WeConnect-cli#53]
Hello,
I get the following error when starting and stopping charging in 0.25.0. Climatisation still works.

```
weconnect-cli --username xxx@xx.xxx --password  yyy set /vehicles/WVWZZZE1ZMPxxxxxx/controls/charging start     
Traceback (most recent call last):
  File "/Users/preventlow/Library/Python/3.8/bin/weconnect-cli", line 8, in <module>
    sys.exit(main())
  File "/Users/preventlow/Library/Python/3.8/lib/python/site-packages/weconnect_cli/weconnect_cli_base.py", line 236, in main
    element.value = args.value
  File "/Users/preventlow/Library/Python/3.8/lib/python/site-packages/weconnect/addressable.py", line 368, in value
    elif issubclass(valType, Enum):
TypeError: issubclass() arg 1 must be a class
```

Using macOS 12.2.1
Rolled back to 0.24.2 and charging works again.

Regards Peter

**Comment by tillsteinbach** [comment]
Can confirm, thanks for reporting!

**Comment by tillsteinbach** [comment]
Can you check if the error is fixed in 0.25.1?

**Comment by Preventlow** [comment]
Starting and stopping charging works now in my setup with 0.25.1 - thanks for the quick fix!
